mods/webui/src/stories/input/Input.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2024 by Fonoster Inc (https://fonoster.com)
+ * http://github.com/fonoster/fonoster
+ *
+ * This file is part of Fonoster
+ *
+ * Licensed under the MIT License (the "License");
+ * you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    https://opensource.org/licenses/MIT
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { StyledInput } from "./Input.styles"
+import MuiTextField from "@mui/material/TextField";
+
+
+const Input = (props: typeof MuiTextField) => <StyledInput {...props} />
```

Comment:
It would be great to hide Mui's implementation by using a custom prop with:

<img width="821" alt="Issue description" src="https://github.com/user-attachments/assets/c00bafce-7279-456d-b347-b869a6849d02" />

Reply:
@psanders but then if we hide the existing props and use custom props we will not be able to use all the features of mui button. I suggest we use the exisiting features of the button and try supporting any custom requirement if any. What you think

Reply:
We want to follow KISS and YAGNI here and only expose the necessary properties.

Reply:
okay